### DISCUSSION TITLE
IP Enforcement

### DISCUSSION THREAD

**Discussion by valcparra** [community/community#77373] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I see that Github supports [Managing IP Addresses](https://docs.github.com/en/enterprise-cloud@latest/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/managing-allowed-ip-addresses-for-your-organization#using-github-actions-with-an-ip-allow-list), however it does not explicitly state that these addresses can be applied to a single user or group of users - ex. admins. Anyone know if this is the case?

**Comment by b8zeek** [comment]
GitHub's feature for managing IP addresses is there to control the access to your organization's data on a network level, not on a user or group level. So, this does not allow you to apply different IP restrictions to the different users or groups within your organization.

The IP restrictions apply uniformly.

If you need to control access on a user or group level, maybe consider using GitHub's role-based access features, e.g. you can assign users to teams, and control each team's access to specific repositories. You can also can have different roles assigned to users in your organization.

Hope this helps you. Good luck. 🍺 

🍀
